### PR TITLE
cfg: fix wrong pd-ctl cmd

### PR DESCRIPTION
**Pull request by glkappe**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by dveeden — COMMENTED** [review]
I think it would be better to use TiUP as it can manage versions, updates, etc. instead of directly calling a binary. But please make sure whatever you do matches the rest of the document

```
$ tiup ctl:v8.3.0 pd cluster
Starting component ctl: /home/dvaneeden/.tiup/components/ctl/v8.3.0/ctl pd cluster
{
  "id": 7423704173585173569,
  "max_peer_count": 3
}
```

**Comment by glkappe** [comment]
> I think it would be better to use TiUP as it can manage versions, updates, etc. instead of directly calling a binary. But please make sure whatever you do matches the rest of the document
> 
> ```
> $ tiup ctl:v8.3.0 pd cluster
> Starting component ctl: /home/dvaneeden/.tiup/components/ctl/v8.3.0/ctl pd cluster
> {
>   "id": 7423704173585173569,
>   "max_peer_count": 3
> }
> ```
> 
> Did the command change or was it always wrong?

1. Just modified the wrong command
2. All the commands in this document are called binary files. This may be a good opportunity to modify them. . .
3. Maybe I can start a new PR to make changes instead of this one, what do you think? @dveeden

**Comment by dveeden** [comment]
@glkappe yes perfect!

**Review by hfxsd — APPROVED** [review]

File: configure-placement-rules.md [hunk 1]
```diff
@@ -211,7 +213,9 @@ cat > rules.json <<EOF
     }
 ]
 EOF
-pd-ctl config placement save --in=rules.json
+
+» ./pd-ctl -u 127.0.0.1:2379 config placement-rules save --in=a.json
```
Comment (hfxsd) [inline configure-placement-rules.md:217]:
```suggestion
» ./pd-ctl -u 127.0.0.1:2379 config placement-rules save --in=rules.json
```

**Pull request merged**